DOCUMENTATION.md

```diff
@@ -21,6 +21,11 @@
   * [Date](#date)
     * [date.isDate](#dateisdate)
     * [date.parseDate](#dateparsedate)
+  * [HttpStatus] (#httpstatus)
+    * [httpStatus.getClass] (#httpstatusgetclass)
+    * [httpStatus.isClientError] (#httpstatusisclienterror)
+    * [httpStatus.isServerError] (#httpstatusisservererror)
+    * [httpStatus.isHttpError] (#httpstatusishttperror)
```

Comment:
Check this menu, there is a space between the title and the link!!!
Use the Visual Studio Preview for Markdown